extensions/2.0/Khronos/KHR_audio/README.md

```diff
@@ -0,0 +1,293 @@
+* K. S. Ernest (iFire) Lee, Individual Contributor
+* Michael Nisbet, Individual Contributor
+* humbletim, Individual Contributor
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+## Status
+
+Open Metaverse Interoperability Group Stage 2 Proposal
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows for the addition of spatialized and non-spatialized audio to glTF scenes.
+
+Audio emitter objects may be added to 3D nodes for positional audio or to the scene for environmental or ambient audio such as background music.
+
+### Example:
+
+```json
+{
+  "extensions": {
+    "KHR_audio": {
+      "sources": [
+        {
+          "name": "environment",
+          "uri": "./environment.mp3"
+        },
+        {
+          "name": "quack",
+          "mimeType": "audio/mpeg",
+          "bufferView": 5
+        }
+      ],
+      "emitters": [
+        {
+          "name": "environment emitter",
+          "type": "global",
+          "gain": 1.0,
+          "loop": true,
+          "playing": true,
+          "source": 0
+        },
+        {
+          "name": "duck emitter",
+          "type": "positional",
+          "gain": 0.8,
+          "loop": false,
+          "playing": false,
+          "source": 1,
+          "positional": {
+            "coneInnerAngle": 6.283185307179586,
+            "coneOuterAngle": 6.283185307179586,
+            "coneOuterGain": 0.0,
+            "distanceModel": "inverse",
+            "maxDistance": 10.0,
+            "refDistance": 1.0,
+            "rolloffFactor": 0.8
+          }
+        }
+      ]
+    }
+  },
+  "scenes": [
+    {
+      "name": "Default Scene",
+      "extensions": {
+        "KHR_audio": {
+          "emitters": [0]
+        }
+      }
+    }
+  ],
+  "nodes": [
+    {
+      "name": "Duck",
+      "translation": [1.0, 2.0, 3.0],
+      "extensions": {
+        "KHR_audio": {
+          "emitter": 1
+        }
+      }
+    }
+  ]
+}
+```
+
+## glTF Schema Updates
+
+This extension consists of two primary data structures: Audio Sources and Audio Emitters. Both sources and emitters are defined on an `KHR_audio` object added to the `extensions`
+ object on the document root.
+
+The extension must be added to the file's `extensionsUsed` array and because it is optional, it does not need to be added to the `extensionsRequired` array.
+
+#### Example:
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    }
+    "extensionsUsed" : [
+        "KHR_audio"
+    ],
+    "scenes": [...],
+    "nodes": [...],
+    "extensions": {
+        "KHR_audio": {
+            "sources": [...],
+            "emitters": [...]
+        }
+    }
+}
+```
+
+### Audio Sources
+
+Audio source objects define audio data to be used in audio emitters. Multiple audio emitters may use the same audio source.
+
+Audio sources can store their data in either a buffer view or reference an external file via uri.
```

Comment:
I think this could also be an inline data URI.

Reply:
The glTF spec defines uris as either relative paths or data uris so I think we can omit that detail.